### ISSUE TITLE
Links don't give you options to assets

### ISSUE THREAD

**Issue by flipxfx** [GraphCMS/Issue-tracker#33]
A lot of times end users want to upload an asset and link to it. Currently they would have to open a new tab with the assets listed, look for the asset and find it's url. It would be nice to be able to choose a link to an asset after clicking the rich text link button in addition to manually entering a url.

**Comment by mlukaszczyk** [comment]
Are you talking about images here or assets in general? Like a PDF? We have a mechanism in place for images but assets of course would make also sense.

**Comment by flipxfx** [comment]
Yeah we would use it mostly for PDFs. But anything that could be uploaded to the assets would be fine. Like if the user wants a link to download an image instead of displaying it etc.

**Comment by mlukaszczyk** [comment]
Got it. We will take this into account.

**Comment by fivetwelve** [comment]
Our site links to quite a few PDFs for tech specs/product information so we're interested in this functionality as well. As an added benefit, it would be more manageable to have our images and PDFs live on the same CDNs.
